filebeat/module/mysql/slowlog/test/mariadb-10.1.21.log-expected.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "@timestamp": "2018-06-13T14:04:36.000Z",
+        "ecs.version": "1.0.0-beta2",
+        "event.dataset": "mysql.slowlog",
+        "event.duration": 2000652000,
+        "event.module": "mysql",
+        "fileset.name": "slowlog",
+        "input.type": "log",
+        "log.flags": [
+            "multiline"
+        ],
+        "log.offset": 24,
+        "mysql.slowlog.current_user": "root",
+        "mysql.slowlog.lock_time.sec": 0.0,
+        "mysql.slowlog.query": "select sleep(2);\n",
+        "mysql.slowlog.query_cache_hit": false,
+        "mysql.slowlog.query_time.sec": 2.000652,
+        "mysql.slowlog.rows_examined": 0,
+        "mysql.slowlog.rows_sent": 1,
+        "mysql.slowlog.thread_id": "5",
+        "mysql.slowlog.timestamp": 1528898676,
```

Comment:
What timestamp is that? Do we need it?

Reply:
This is the timestamp when the query happened, it is used to obtain `@timestamp`. It could be actually removed after that, but is not added by this change. I could remove it in another PR.

Reply:
SGTM